requirements-dev.txt

```diff
@@ -10,4 +10,4 @@ mypy
 mypy-extensions
 types-requests
 types-PyYAML
-tox
\ No newline at end of file
+tox
```

Comment:
why's it say there's a change in the deps?

Reply:
might be an empty line?